src/VsixV3/Microsoft.Net.Sdk.swr

```diff
@@ -0,0 +1,32 @@
+use vs
+
+package name=Microsoft.Net.Sdk
+        version=$(Version)
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="Microsoft Net Sdk"
+                       description="SDK for .NET projects"
+
+folder "InstallDir:MSBuild\Sdks\Microsoft.Net.Sdk\sdk"
```

Comment:
I think the Sdk name in the folder path should be lowercased (ie `microsoft.net.sdk`).  This is what NuGet does, and is how you can have case-insensitive package or Sdk names on a case sensitive filesystem.  See also Microsoft/MSBuild#1421